src/HieDb/Run.hs

```diff
@@ -127,10 +129,25 @@ optParser defdb colr
   <*> switch (long "reindex" <> short 'r' <> help "Re-index all files in database before running command, deleting those with missing '.hie' files")
   <*> switch (long "keep-missing" <> help "Keep missing files when re-indexing")
   <*> optional (strOption (long "src-base-dir" <> help "Provide a base directory to index src files as real files"))
+  <*> skipFlags
   where
     colourFlag = flag' True (long "colour" <> long "color" <> help "Force coloured output")
             <|> flag' False (long "no-colour" <> long "no-color" <> help "Force uncoloured ouput")
             <|> pure colr
+    skipFlags = do
+        refs <- switch (long "skip-refs" <> help "Skip refs table when indexing")
+        decls <- switch (long "skip-decls" <> help "Skip decls table when indexing")
+        defs <- switch (long "skip-defs" <> help "Skip defs table when indexing")
+        exports <- switch (long "skip-exports" <> help "Skip exports table when indexing")
+        typeRefs <- switch (long "skip-typerefs" <> help "Skip typerefs table when indexing")
+        pure $ SkipOptions
```

Comment:
Applicative do intentionally chosen here since it makes this less fragile to changing the argument order of `SkipOptions`